universal-application-tool-0.0.1/app/auth/saml/SamlCiviFormProfileAdapter.java

```diff
@@ -155,6 +182,13 @@ public CiviFormProfileData mergeCiviFormProfile(
     }
     String emailAddress = saml2Profile.getEmail();
     civiFormProfile.setEmailAddress(emailAddress).join();
+
+    Optional<String> authorityId = getAuthorityId(saml2Profile);
```

Comment:
Suggestion: we can do this streamlined.  I fixed this up of OIDC in the PR I sent you

```
authorityId = getAuthorityId(saml2Profile).orElseThrow(
    () -> new InvalidSamlProfileException("Unable to get authority ID from profile.") );
```

and remove the optional .